src/main/java/uk/gov/gchq/palisade/rule/Rules.java

```diff
@@ -20,6 +20,7 @@
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 
+import uk.gov.gchq.palisade.Generated;
 import uk.gov.gchq.palisade.ToStringBuilder;
 import uk.gov.gchq.palisade.jsonserialisation.JSONSerialiser;
 
```

Comment:
update setters and getters, add @generated. 
Use setters within mutator methods

update equals, hashCode and toString methods

Reply:
Care needs to be taken with the equals method here and might benefit from unit testing and therefore not having the generated annotation. But it still needs the hashcode and tostring updating. Also the setters, getters and mutators that follow the standard pattern should be updated as @developer01189998819991197253 mentioned above.